modules/apps/object/object-rest-test/src/testFunctional/tests/SupportManyToManyRelationships.testcase

```diff
@@ -13,19 +13,20 @@ definition {
 	}
 
 	tearDown {
-		if (${testPortalInstance} == "true") {
-			PortalInstances.tearDownCP();
-		}
-		else {
-			ObjectDefinitionAPI.modifyRelationship(
-				deletionType = "cascade",
-				objectName = "University");
+		ObjectDefinitionAPI.modifyRelationship(
```

Comment:
Thanks for adding tearDownCp/tearDownCPNoSelenium in some of the files. Just wondering if we use tearDownCP..., then all portal instance will be deleted, so I think we don't need to move the following from else condition. No sure if my understanding is correct, any ideas @magjed4289 @Hijky ?

Reply:
@yanan if we use tearDownCP it should delete everything from the test but that is not always the case. also, the teardownCP would work in CI. Locally, if tests running without creating a new instance items need to be deleted separately.